appholder/src/main/java/com/android/identity/wallet/HolderApp.kt

```diff
@@ -87,6 +89,17 @@ class HolderApp: Application() {
             secureAreaRepository.addImplementation(softwareSecureArea)
             return CredentialStore(storageEngine, secureAreaRepository)
         }
+
+        /**
+         * Create a PresentationLogStore
+         */
```

Comment:
Create a Store by following suit of other stores being created in the Holder app.